src/components/TableFilter.js

```diff
@@ -148,138 +143,128 @@ class TableFilter extends React.Component {
     this.props.onFilterUpdate(index, event.target.value, 'textField');
   };
 
-  renderCheckbox(columns) {
+  renderCheckbox(column, index) {
     const { classes, filterData, filterList } = this.props;
 
-    return columns.map((column, index) =>
-      column.filter ? (
-        <div className={classes.checkboxList} key={index}>
-          <FormGroup>
+    return (
+      <GridListTile key={index} cols={2}>
+        <FormGroup>
+          <Grid xs={12}>
             <Typography variant="body2" className={classes.checkboxListTitle}>
               {column.label}
             </Typography>
+          </Grid>
+          <Grid container>
             {filterData[index].map((filterColumn, filterIndex) => (
-              <FormControlLabel
-                key={filterIndex}
-                classes={{
-                  root: classes.checkboxFormControl,
-                  label: classes.checkboxFormControlLabel,
-                }}
-                control={
-                  <Checkbox
-                    className={classes.checkboxIcon}
-                    onChange={this.handleCheckboxChange.bind(null, index, filterColumn)}
-                    checked={filterList[index].indexOf(filterColumn) >= 0 ? true : false}
-                    classes={{
-                      root: classes.checkbox,
-                      checked: classes.checked,
-                    }}
-                    value={filterColumn != null ? filterColumn.toString() : ''}
-                  />
-                }
-                label={filterColumn}
-              />
+              <Grid item>
```

Comment:
Here at line 159 the console is saying 
```
Warning: Failed prop type: The property `xs` of `Grid` must be used on `item` ...
```
Can you change this markup to `<Grid item xs>` please

Reply:
Done